junit5-support/src/main/kotlin/io/specmatic/test/reports/OpenApiCoverageReportProcessor.kt

```diff
@@ -22,9 +23,12 @@ class OpenApiCoverageReportProcessor (private val openApiCoverageReportInput: Op
     }
 
     override fun process(specmaticConfig: SpecmaticConfig) {
-        val reportConfiguration = specmaticConfig.report!!
+        val reportConfiguration = getReport(specmaticConfig)!!
```

Comment:
getReport should not be used directly as it's on the companion object. There are already methods on SpecmaticConfig for fetching the needed data and those should be used directly.